netcore/NetCore.Test/xunit.runner.json

```diff
@@ -0,0 +1,5 @@
+{
+  "methodDisplay": "classAndMethod",
+  "parallelizeAssembly": true,
+  "parallelizeTestCollections": true
+}
```

Comment:
Missing terminating new lines in multiple files.

Reply:
Fixed.